data/amd/register/msr/MSR0000_0010.yml

```diff
@@ -0,0 +1,18 @@
+- name: MSR0000_0010
+  long_name: "Time Stamp Counter"
+  purpose: |
+       "
+       The TSC uses a common reference for all sockets, cores and threads. 
+       "
+  size: 64
+  arch: x86
+
+  access_mechanisms:
+      - name: rdmsr
+        is_read: True
+        address: 0x4c1
+
+      - name: wrmsr
+        is_write: True
+        address: 0x4c1
+
```

Comment:
There are no fieldsets or fields defined for this MSR. According to the manual, it should contain one fieldset + 1 field named TSC (time stamp counter), that is 64 bits wide.